### ISSUE TITLE
HK: Check Ty Gibson Client numbers for possible discrepancies -> TEST DOCS

### ISSUE THREAD

**Issue by jesusitd** [intakedesk/PowerBI-General#254]
![image](https://user-images.githubusercontent.com/50643267/64717917-2fdffc00-d493-11e9-9f3d-dd23dc43e18b.png)

![image](https://user-images.githubusercontent.com/50643267/64717861-10e16a00-d493-11e9-869c-b3a2d27da667.png)

![image](https://user-images.githubusercontent.com/50643267/64717873-18087800-d493-11e9-9c78-6509877b9fd6.png)

No discrepancies are evident. Waiting on HK for revision.

**Comment by jesusitd** [comment]
Mistery solved. There were 2 test retainers found for deleted cases:
```sql
SELECT      documents.`id` AS `document_id`, 
        documents.`date_entered`, 
        documents.`document_name`, 
        documents.`status_id`, 
        documents.`document_revision_id`,
        documents.`template_type`, 
        l.first_name, l.last_name,
        documents_cstm.`aos_pdf_templates_id_c` AS `template_id`
        , dc.case_id
        , c.case_number
        , c.name
,        c.deleted
FROM        documents 
INNER JOIN        documents_cstm ON documents.id=documents_cstm.id_c AND aos_pdf_templates_id_c = 'e3aa760e-25d7-ed4a-84e3-5d16286585e2'
left JOIN leads_documents_1_c ld ON documents.id = ld.leads_documents_1documents_idb -- AND ld.leads_documents_1leads_ida = '92a040bb-d75a-2205-604f-58821d3f80aa'
LEFT JOIN leads l ON ld.leads_documents_1leads_ida = l.id AND l.deleted = 0
LEFT JOIN documents_cases dc ON documents.id = dc.document_id
LEFT JOIN cases c ON dc.case_id = c.id -- AND c.deleted = 0
WHERE       documents.deleted = 0 AND status_id = 'Complete'
```

**Comment by jesusitd** [comment]
**Now, need to hide Test Data from clients reports.**

**Comment by jesusitd** [comment]
Test Docs filter will be tackled in a separate issue.
